app/src/main/java/com/connorlay/knitgrid/activities/PatternGridViewActivity.java

```diff
@@ -0,0 +1,55 @@
+package com.connorlay.knitgrid.activities;
+
+import android.os.Bundle;
+import android.support.v4.app.FragmentTransaction;
+import android.support.v7.app.AppCompatActivity;
+import android.util.Log;
+
+import com.connorlay.knitgrid.R;
+import com.connorlay.knitgrid.fragments.PatternHighlightFragment;
+import com.connorlay.knitgrid.models.Stitch;
+import com.connorlay.knitgrid.presenters.PatternPresenter;
+
+import butterknife.ButterKnife;
+
+public class PatternGridViewActivity extends AppCompatActivity{
```

Comment:
This class should not be here. It was replaced with `PatternDetailActivity` in earlier commits.